joomla/joomla-cms
# [4.1][bugfix] Fix language inheritance for child template

Pull Request for Issue #36218 .

### Summary of Changes
- Load the language files similar to https://github.com/joomla/joomla-cms/blob/40eff93677773b400fddcc4123af06a7216eec52/administrator/components/com_templates/src/Model/StyleModel.php#L420-L425 so that inheritance is followed correctly.


### Testing Instructions
Goto /administrator/index.php?option=com_templates&view=template and check the description for a child template


### Actual result BEFORE applying this Pull Request

Weird text

### Expected result AFTER applying this Pull Request

Readable text

### Documentation Changes Required

No, this is a bug
